### ISSUE TITLE
doi via zenodo? 

### ISSUE THREAD

**Issue by lheagy** [jupyter4edu/jupyter-edu-book#158]
Would folks be supportive of archiving this repository on [zenodo](http://zenodo.org)? This would provide a doi which would be helpful when citing it. Thoughts?

**Comment by moorepants** [comment]
Yes, but we never seemed to reach a version 1.0 (first draft), or have we?

**Comment by labarba** [comment]
No, we didn't. In fact, some of the open issues are major, in my opinion.

**Comment by lheagy** [comment]
Thats fair. Would it make sense to archive the current version under an 0.1? I appreciate that we don't want to signal that it is at a 1.0 state, but it is still a useful resource as is, and it might be worth putting a marker on the work that has been achieved to date?

**Comment by zingale** [comment]
I think that's a good idea.

**Comment by dsblank** [comment]
Maybe version higher than 0.1? We're almost "done"! :)

**Comment by elliewix** [comment]
We did a three day sprint, so why not .3?
